docs/configuration.md

```diff
@@ -175,7 +175,7 @@ Namespace Isolation is disabled by default.
 
 #### Usage example
 
-Let's setup an example where we:
+Lets setup an example where we:
```

Comment:
Remove this change. The others are good, but "let's" is a contraction of "let us" so it uses an apostrophe.